nixos/modules/config/pulseaudio.nix

```diff
@@ -160,6 +167,31 @@ in {
             if activated.
           '';
         };
+
+        resample-method = mkOption {
+          type = types.str;
```

Comment:
@peterhoeg I think it even might vary depending on different pulseaudio packages (and if it is just in the future).